aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -75,17 +75,17 @@ Tensor nested_from_padded_cuda(
             offsets_ptr,
             input_size_ptr,
             output_size_ptr,
-            padded_contiguous.dim() - 2,
-            padded_contiguous.sizes()[0]);
+            static_cast<int>(padded_contiguous.dim() - 2),
+            static_cast<int>(padded_contiguous.sizes()[0]));
```

Comment:
didn't we fix this seperately in another PR?

Reply:
Due to rebase conflicts... Fixed